keras/layers/core/dense_test.py

```diff
@@ -8,8 +8,11 @@
 
 
 class DenseTest(testing.TestCase):
+    """Test the Dense layer."""
```

Comment:
When adding docstrings or comments, make sure that they provide information that is not redundant with the information provided by the code and in particular by naming choices. Here, `DenseTest` is already sufficient as a name.

Reply:
Thank you for your feedback.